docs/en/cookbook/accessing-private-properties-of-the-same-class-from-different-instance.rst

```diff
@@ -0,0 +1,74 @@
+Accessing private/protected properties/methods of the same class from different instance
+============================
+
+.. sectionauthor:: Michael Olsavsky (olsavmic)
+
+As explained in the
+`restrictions for entity classes in the manual <https://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/architecture.html#entities>`_,
```

Comment:
You should use a target instead of using a link to the html file: https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets